### PR TITLE
fix(api-doc-gen): remove field 'raw' from api-doc-file

### PR DESCRIPTION
**Pull request by tskdsb**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @iawia002 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by tskdsb** [comment]
生成文档的体积减小 98.6%
image

File: utils/generators/swagger/generator.go [hunk 1]
```diff
@@ -271,7 +271,8 @@ func (g *Generator) schemaForStruct(typ *api.Type) *spec.Schema {
 		schema.SetProperty(name, *fieldSchema)
 	}
 	g.schemas[key] = schema
-	ref.ExtraProps = map[string]interface{}{rawSchemaKey: schema}
+	// This will cause big-size of api-doc-file
+	// ref.ExtraProps = map[string]interface{}{rawSchemaKey: schema}
```
Comment (iawia002) [inline utils/generators/swagger/generator.go:275]: We can just delete instead of commenting out these

Comment (tskdsb) [reply utils/generators/swagger/generator.go]: ok

**Pull request merged**